include/SFML/Graphics/Text.hpp

```diff
@@ -224,14 +270,47 @@ class SFML_GRAPHICS_API Text : public Drawable, public Transformable
     Uint32 getStyle() const;
 
     ////////////////////////////////////////////////////////////
-    /// \brief Get the global color of the text
+    /// \brief Get the fill color of the text
+    ///
+    /// \return Fill color of the text
+    ///
+    /// \see setFillColor
+    ///
+    /// \deprecated This function is deprecated and may be removed in future releases.
+    /// Use getFillColor instead.
```

Comment:
I changed it to "There is now fill and outline colors instead of a single global color."